src/sentry/features/base.py

```diff
@@ -62,3 +63,14 @@ def __init__(self, name: str, user: User) -> None:
 
     def get_subject(self) -> User:
         return self.user
+
+
+class FeatureHandlerStrategy(Enum):
+    """
+    This controls whether the feature flag is evaluated statically,
+    or if it's managed by a remote feature flag service.
+    See https://develop.sentry.dev/feature-flags/
+    """
+
+    INTERNAL = 1
+    REMOTE = 2
```

Comment:
Would be nice if we had comments on these?

Reply:
will f/u